.gitignore

```diff
@@ -1 +1,9 @@
 /target
+
+.env
+
+# Hornet files
+/alphanet
+config.json
+peering.json
+profiles.json
```

Comment:
Why are those excluded here? :thinking: Shouldn't inx-chronicle be independent of the concrete node impl, or is this just temporary?